JapaneseFilter/sections/specific.txt

```diff
@@ -44,6 +44,7 @@ iine-kunitachi.net,2d-erocafe.com,acueducto.jp,akashi-journal.com,analsexgay.net
 !
 !
 makitani.net##.ablock
+||image1.rank-king.jp/item/image/original/8408712.jpg
```

Comment:
The image path is not likely to be stable, better `ranking.goo.ne.jp###sticky_adsense`

Reply:
ok